parts/linux/cloud-init/artifacts/krustlet.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Krustlet
+
+[Service]
+Restart=on-failure
+RestartSec=5s
+Environment=KUBECONFIG=/var/lib/kubelet/kubeconfig
+Environment=KRUSTLET_CERT_FILE=/etc/krustlet/config/krustlet.crt
```

Comment:
We probably want to use the kubelet server TLS certs here: https://github.com/Azure/AgentBaker/blob/f91e734bdad2b692dd03d5dc3b90a3ad68d8d590/parts/linux/cloud-init/artifacts/cse_config.sh#L82-L83

The certs should be created with CSE, so we can just use those paths as-is